test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -5333,7 +5357,7 @@ TEST_F(AtenXlaTensorTest, TestIndexFillWithScalar) {
       });
 
       ExpectCounterNotChanged("aten::.*", cpp_test::GetIgnoredCounters());
-      ExpectCounterChanged("xla::index_fill_", cpp_test::GetIgnoredCounters());
+      ExpectCounterChanged("xla::index_fill", cpp_test::GetIgnoredCounters());
```

Comment:
For all the operations in this PR, we test one version of the call. Though, to be clear, this means some code in `aten_xla_type.cpp` doesn't have coverage on the tests. 

I don't have a good solution for better coverage without the large cost for expanding test for all such cases. Generally, I am ok with it since implementations are very similar.